plugins/discovery-azure-classic/src/main/java/org/apache/commons/lang/StringUtils.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.apache.commons.lang;
+
+import java.util.Collection;
+import java.util.Iterator;
+
+/**
+ * Minimal shim to satisfy Azure Classic which expects commons-lang 2.x.
+ * Delegates to commons-lang3.
+ */
```

Comment:
Thoughts on this? It creates a shim around StringUtils from commons-lang3 that spoofs that this is coming from commons-lang:commons-lang. The alternative is upgrading azure deps, but that is non-trivial.

Reply:
[This](https://mvnrepository.com/artifact/com.microsoft.azure/azure-core/0.9.3) appears to be the azure dependency this plugin is using? Which is from February 2016? Adding this StringUtils wrapper hardly seems like the biggest problem here...

Reply:
I would definitely support taking it out of the core repo and create a separate repo as https://github.com/opensearch-project/OpenSearch/issues/17246 proposed.